src/client/feed/Feed.js

```diff
@@ -2,16 +2,47 @@ import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import moment from 'moment';
 
-const Feed = ({ feed }) => (
-  <div>
-    <h3>Search results</h3>
-    <div className="ui divided items">
+const Feed = ({ feed, hiddenWords }) => {
+  const filteredFeed = filterPostsForFeed(feed, hiddenWords);
+
+  return (
+    <div>
+      <h3>Search results, showing {filteredFeed.length} posts.</h3>
+      (Hiding {feed.length - filteredFeed.length} posts)
+      <div className="ui divided items">
+        {
+          filteredFeed
+            .map((feedItem) => (<FeedItem content={feedItem}/>))
+        }
+      </div>
+    </div>
+  );
+}
+
+const filterPostsForFeed = (feed, hiddenWords) => {
+  return feed.filter((feedItem) => {
+    switch (feedItem.source) {
+      case 'twitter':

```

Comment:
Why is this twitter only ? wouldn't this filtering of post work for every source ?